src/jdk.compiler/share/classes/com/sun/tools/javac/comp/Flow.java

```diff
@@ -747,23 +748,25 @@ private Set<Symbol> coveredSymbolsForCases(DiagnosticPosition pos,
 
         private Set<Symbol> coveredSymbols(DiagnosticPosition pos, Type targetType,
                                            Iterable<? extends JCCaseLabel> labels) {
-            Set<Symbol> constants = new HashSet<>();
-            Map<Symbol, List<JCDeconstructionPattern>> categorizedDeconstructionPatterns = new HashMap<>();
+            Set<Symbol> coveredSymbols = new HashSet<>();
+            Map<Symbol, List<JCRecordPattern>> deconstructionPatternsBySymbol = new HashMap<>();
```

Comment:
since you seem to have settled on "recordPattern" for implementation names - you can probably revisit some of these names to say "record" instead of "deconstruction".

Reply:
Right. Will do.